### ISSUE TITLE
learning_rate

### ISSUE THREAD

**Issue by RuiboFan** [ujscjj/DPTNet#7]
how to set the  training learning rate?

**Comment by ujscjj** [comment]
It's here: https://github.com/ujscjj/DPTNet/blob/master/others/optimizer_dptnet.py
